task_1/main.cpp

```diff
@@ -0,0 +1,211 @@
+#include <string.h>		
+#include <stdlib.h>
+#include <stdio.h>
+
```

Comment:
Add these two commands to make code clearer
using String = char*;
using List = char **;

Reply:
Only string.h and stdlib.h can be used in this file

Reply:
I suggest to use c++ style **reinterpret_cast<Type>(value)** instead of a c-style casting **(Type) value**.